pydantic_xml/serializers/factories/primitive.py

```diff
@@ -67,10 +67,12 @@ class AttributeSerializer(Serializer):
     def from_core_schema(cls, schema: PrimitiveTypeSchema, ctx: Serializer.Context) -> 'AttributeSerializer':
         namespaced_attrs = ctx.namespaced_attrs
         name = ctx.entity_path or ctx.field_alias or ctx.field_name
-        ns = ctx.entity_ns or (ctx.parent_ns if namespaced_attrs else None)
+        ns = select_ns(ctx.entity_ns, ctx.parent_ns if namespaced_attrs else None)
         nsmap = merge_nsmaps(ctx.entity_nsmap, ctx.parent_nsmap)
         computed = ctx.field_computed
 
+        if ns == '':
+            raise errors.ModelFieldError(ctx.model_name, ctx.field_name, "attributes with default namespace are forbidden")
```

Comment:
please split the line to conform the flake format:
```
pydantic_xml/serializers/factories/primitive.py:75:121: E501 line too long (123 > 120 characters)
```

Reply:
Done! Appreciate all your help 😄